src/gui_common/TranslationSiteButton.cs

```diff
@@ -0,0 +1,14 @@
+using Godot;
+using System;
+
+/// <summary>
+///   Shows a version label
+/// </summary>
+public class TranslationSiteButton : Button
```

Comment:
Unless this button is planned to be planned to be used somewhere else, I would just copy it into the options menu instead of instancing it, and then just adding the correct signal